plasmapy/tests/test_requirements.py

```diff
@@ -0,0 +1,141 @@
+"""Tests for the consistency of requirements."""
+
+import os
+import pytest
+import setuptools
+import tomli
+
+from typing import Dict, Set
+
+import plasmapy
+
+import pathlib
+base_directory = pathlib.Path(__file__).parent.parent
```

Comment:
`__file__` is `test_requirements.py`, whose parent is `tests`, whose parent is `plasmapy`, which means we're missing one more parent, or, more succinctly:
```suggestion
base_directory = pathlib.Path(__file__).parents[2]
```